src/modbus.c

```diff
@@ -369,6 +376,19 @@ static int mb_init_connection(mb_host_t *host) /* {{{ */
       ERROR("Modbus plugin: Creating new Modbus/RTU object failed.");
       return -1;
     }
+    switch (host->uarttype) {
+    case UARTTYPE_RS485:
+      if (modbus_rtu_set_serial_mode(host->connection, MODBUS_RTU_RS485))
+        DEBUG("Modbus plugin: Setting RS485 mode failed.");
+    /* fall-through */
```

Comment:
Use of `/* fall-through */` looks strange here.

Reply:
The `modbus_rtu_set_serial_mode` appeared only in  libmodbus-2.9.4 and available only on Linux platform.
You need to add needed conditionals to handle this correctly.

Reply:
I changed them to 'break' instead, it doesn't really matter at all in that place obviously.